styles.js

```diff
@@ -7,7 +7,28 @@ export default createGlobalStyle`
     box-sizing: border-box;
   }
   body {
-    margin: 0;
+    margin: 0 auto;
     font-family: system-ui;
   }
+
+  li {
+    list-style-type:none;
+  }
+  
+  h1{
+    font-size:1.5rem;
+  }
+
+  p{
+    margin:5px;
+  }
```

Comment:
Looks like Prettier is not working properly on save.

If you set the `list-style-type` to `none`, your list should have the `role="list"` to be accessible in Safari as well.